src/Worker/AzureManaged/DurableTaskSchedulerWorkerOptions.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.ComponentModel.DataAnnotations;
+using Azure.Core;
+using Azure.Identity;
+using Grpc.Core;
+using Grpc.Net.Client;
+
+namespace Microsoft.DurableTask;
+
+/// <summary>
+/// Options for configuring the Durable Task Scheduler.
+/// </summary>
+public class DurableTaskSchedulerWorkerOptions
```

Comment:
To avoid code duplication, but also keep these types separate, I wonder if you can create an internal `DurableTaskSchedulerOptions` in the shared project, have all of the properties public and then inherit it here?

I am not 100% sure that will work, but worth a shot if you want. If not then it is fine to keep it duplicated.

Reply:
discussed with Chris, plan to have this as a future todo